content/en/docs/ops/diagnostic-tools/multicluster/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Troubleshooting Multi-Cluster
+description: Describes tools and techniques to diagnose issues with multi-cluster and multi-network installations.
+weight: 20
+keywords: [debug,multi-cluster,multi-network,envoy]
+owner: istio/wg-environments-maintainers
+test: n/a
```

Comment:
I'm not sure if this should be a `no`. If there is anything that can be tested on this page (and I suspect there is as there are a number of scripts), this should be `no`. `n/a` is reserved for those pages that do not need tests.

Reply:
I guess testing this could be possible: intentionally setting up clusters the wrong way then verifying the results are broken in the ways we expect?